internal/pkg/pkg.go

```diff
@@ -43,6 +44,11 @@ var DeprecatedKptfileVersions = []string{
 	"v1alpha1",
 }
 
+var SupportedKptfileVersions = []string{
+	kptfilev1alpha2.KptFileVersion,
```

Comment:
So we are keeping both version of the types. Do we use the v1alpha1 type anywhere? It seems like we allow reading with both types, but how do we handle writing back the Kptfile? Do we automatically update the Kptfile? If so, does that in any way impact functionality that relies on checking diffs?

Reply:
I think we are not using `v1alpha1` anywhere. Where do we need to write back the Kptfile?

Reply:
removed `v1alpha2`